python/src/dpsim/matpower.py

```diff
@@ -1,38 +1,32 @@
 import pandas as pd
 import numpy as np
 import scipy.io
-
+import sys
+sys.path.append('/dpsim/build')
```

Comment:
This looks like a hack to make it work without installing dpsim or adding dpsim to pythonpath. It only works in the dev container I assume.

Reply:
```python
import sys
from os.path import dirname as dn

dpsim_dir = dn(dn(dn(dn(__file__))))
sys.path.append(dpsim_dir)
```

Should be more robust?

Reply:
@m-mirz This was added when developing the Matpower reader python module exactly for the two reasons you mention.

Reply:
I discussed this with @stv0g  and we concluded that the best solution would be to not include any workaround like this. Please just update your pythonpath when developing… it is just copy & paste from the docs.